src/index.ts

```diff
@@ -38,9 +43,9 @@ function getReport(
 
   const coverageDiff = headTypeCoverage - baseTypeCoverage;
 
-  const shortDescription = `Change: ${renderSign(coverageDiff)}${perc(coverageDiff)} Total: ${perc(
-    headTypeCoverage,
-  )} ${newTypedSymbolsNote}`;
+  const shortDescription = `Change: ${renderSign(coverageDiff)}${perc(coverageDiff)}${
+    options.atLeast === undefined ? "" : ` (limit ${perc(options.atLeast)})`
```

Comment:
Maybe we need to re-format this message somehow.

Reply:
Yeah I will refactor it to separate function as I also wanted to get rid of showing how many typed symbols were added since its quite useless metric and we should care only about untyped symbols.